js&css/web-accessible/functions.js

```diff
@@ -3,14 +3,15 @@
 --------------------------------------------------------------*/
 
 ImprovedTube.childHandler = function (node) {
-	if (node.nodeName !== 'SCRIPT' && node.nodeName !== 'iron-iconset-svg' && node.nodeName !== 'svg' && node.nodeName !== '#text'&& node.nodeName !== '#comment' && node.nodeName !== 'SPAN' && node.nodeName !== 'DOM-IF' && node.nodeName !== 'DOM-REPEAT') {
-		var children = node.children;
-		this.ytElementsHandler(node);
+	if (node.nodeName === 'SCRIPT' || node.nodeName === 'iron-iconset-svg' || node.nodeName === 'svg' || node.nodeName === '#text' || node.nodeName === '#comment' || node.nodeName === 'SPAN' || node.nodeName === 'DOM-IF' || node.nodeName === 'DOM-REPEAT') {
```

Comment:
Everything in 10aef10e9070d7781166887416dc46feb008d401 is pure semantics and doesnt do anything. Browser js engine will optimize it to same logical form when loading the page.

Reply:
yes (`!==` .. `&&`   vs  `===` .. `||`   just improves readability (might save 2ns per cycle) )
but of course 'childHandler' is critical

> everything

besides? `if (node.getAttribute('itemprop'))  ) { ...`

Reply:
yes (`!==` .. `&&`   vs  `===` .. `||`   just improves readability (might save 2ns per cycle) )
but of course 'childHandler' is critical

> everything

besides `if (node.getAttribute('itemprop'))  ) { ...`